CRVResponse/test/wideband/wideband4modules.fcl

```diff
@@ -108,7 +108,7 @@ physics.producers.CrvSteps.stepPointsModuleLabels   : ["g4run"]
 physics.producers.CrvPhotons.crvStepModuleLabels    : ["CrvSteps"]
 physics.producers.CrvPhotons.crvStepProcessNames    : ["CrvWideband4ModulesSim"]
 physics.producers.CrvPhotons.photonYieldVariationScale    : 0.02   //gaussian distribution with sigma equal to 2.0% of the nominal photon yield
-physics.producers.CrvPhotons.photonYieldScaleFactor       : 1.00
+physics.producers.CrvPhotons.photonYieldScaleFactor       : 0.92   //0.92 seems to give the best match with Wideband data
```

Comment:
How consistent is this value with MDC2020 outputs? What production stage would we need to rerun to be consistent with the wideband data?

Reply:
Our knowledge about the light yields have evolved over time (and are still evolving). Updating the light yield requires re-runing the digitization stage.